core/src/components/item/test/disabled/index.html

```diff
@@ -63,6 +67,12 @@
             <ion-checkbox disabled checked> Disabled Checkbox </ion-checkbox>
           </ion-item>
 
+          <ion-radio-group value="radio">
+            <ion-item>
+              <ion-radio disabled value="radio"> Disabled Radio </ion-radio>
+            </ion-item>
+          </ion-radio-group>
+
```

Comment:
`radio/test/states` does not check the appearance of a disable radio in an `ion-item` (if that makes a difference).

Reply:
This and the textarea were really just added because you couldn't see them all side by side like you can with the other form controls (checkbox, input, select, etc.)

Reply:
I think these are patterns we should be moving away from in favor of taking screenshots of individual states. We take a screenshot of the entire page here: https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/item/test/disabled/item.e2e.ts this makes it hard to know what exactly we are testing. The test implies that there are behaviors specific to the item when disabled form controls are placed inside. However, the disabled state for modern form controls such as radio and textarea are not coupled to the item, so they don't change even if they are in an item.

Reply:
I think item is just in a weird in between state right now. Having the checkbox & radio in the legacy item disabled test allowed me to catch this that I missed when originally updating the styles: https://github.com/ionic-team/ionic-framework/pull/27396/commits/1f27e945fe485fe1c318390d83f83f2e753ba3b3

I can see why we don't need them in the modern item disabled test, but I think they were just copied over from the legacy test. I don't think adding these does any harm since the test already exists and the screenshot would've been updated either way, but I can remove them if you want.

Reply:
I agree that having this in the legacy test makes sense since we do not screenshot legacy disabled controls individually. For the modern controls we really should not even have `item/test/disabled` since these states are already captured on each component. I don't mind if we keep the tests, but I also don't think adding these tests gives us additional test coverage since we already test these states in each component.